Recommend `make -C test-suite` over other methods. (#14698)
coq/coq: test-suite/README.md

# Coq Test Suite

The test suite can be run from the Coq root directory by `make test-suite`.

> However, for incremental test-suite builds, we recommend running `make` directly

Reviewer: ```suggestion
However, for incremental test suite builds, we recommend running `make`
```

from the test-suite directory (or with `make -C test-suite`).

> This Makefile is compatible with both a full Dune build
> or a legacy hybrid build.

Reviewer: ```suggestion
This Makefile is compatible with both Dune
and legacy hybrid builds.
```
